packages/ui/build/rollup/configs/styles.js

```diff
@@ -4,26 +4,35 @@ import deleteJunkPlugin from '../plugins/rollup-delete-junk'
 import transformScssPlugin from '../plugins/rollup-transform-scss'
 import copyPlugin from 'rollup-plugin-copy'
 import postcssImport from '../postcss-plugins/postcss-import'
+import vuePlugin from 'rollup-plugin-vue'
+import { nodeResolve as nodeResolvePlugin } from '@rollup/plugin-node-resolve'
+import typescriptPlugin from 'rollup-plugin-typescript2'
+import commonjsPlugin from '@rollup/plugin-commonjs'
 
-/** Used for tree-shaking. It creates separate modules in ESM format, that can be tree-shakable by any bundler. */
 export function createStylesConfig ({ input, outDir = 'dist/', minify = false }) {
   const inputPathWithoutFilename = input.split('/').slice(0, -1).join('/')
-
   const transformSrc = (src) => `./${src.replace(inputPathWithoutFilename, '')}`
 
   return defineConfig({
-    input,
+    input: ['./src/main.ts', input],
```

Comment:
`'./src/main.ts'` already in input argument in defaultParamss..

```suggestion
    input,
```